src/server/controllers/comment.controller.ts

```diff
@@ -78,6 +80,41 @@ export const upsertCommentHandler = async ({
 }) => {
   try {
     const { ownerId, locked } = ctx;
+    const { modelId } = input;
+
+    // Get model and 1 version to check if user has access:
+    const model = await dbRead.model.findUnique({
+      where: { id: modelId },
+      select: {
+        id: true,
+        modelVersions: {
+          take: 2,
```

Comment:
In the comment above you mentioned 1 version, but here it says 2, which is it? 😛